### DISCUSSION TITLE
When pressing space to skip typewriting,  the signals are ignored in ver. 1.4

### DISCUSSION THREAD

**Discussion by fire7side** [dialogic-godot/dialogic#773] [General]
Mostly I'm using the talk signal, so I can just pick it up in Godot, but if something else were in there it would get skipped.

**Comment by Jowan-Spooner** [comment]
What signals are you talking about? The [signal] command?

**Comment by fire7side** [comment]
Yes.  I send a signal with it and it works fine, but if I press space bar
or enter and the typewriter text stops and shows the full sentence, then it
no longer gives a signal.

**Reply by Jowan-Spooner** [reply]
That makes sense, you are skipping the command, so it never looks up that thing. I agree, that this might be bad behaviour, but before changing anything, we must figure out, what the expected behaviour really should be. Signal seems to be the only command that should possibly be executed even when text is skipped, while [speed] and [pause] do nothing when you skip. Not sure about [play] either. it could see people wanting the sounds to only play at a specific point, but also want it to play definitely, even if skipped. 

I need to think about this a bit longer.
